astropy/units/tests/test_format.py

```diff
@@ -856,7 +856,6 @@ def test_celsius_fits():
     "format_spec, string",
     [
         ("generic", "dB(1 / m)"),
-        ("unscaled", "dB(1 / m)"),
```

Comment:
This was added only in #14413, so no big loss to remove it. Otherwise, "unscaled" was not used at all!